ui/imports/shared/stores/CommunityTokensStore.qml

```diff
@@ -116,11 +116,13 @@ QtObject {
     }
 
     // Airdrop tokens:
-    function airdrop(communityId, airdropTokens, addresses) {
+    function airdrop(communityId, airdropTokens, addresses, feeAccountAddress) {
+        // TODO: Take `feeAccountAddress` into account for the airdrop
```

Comment:
Isn't there some backend code already for this?

Reply:
From what I know not, I will discuss it with @endulab next week when he's back from vacations.